--- postgresql.conf.org	2018-02-13 17:31:23.655661473 +0900
+++ postgresql.conf	2018-02-13 17:32:08.826838961 +0900
@@ -56,11 +56,11 @@
 
 # - Connection Settings -
 
-#listen_addresses = 'localhost'		# what IP address(es) to listen on;
+listen_addresses = '*'		# what IP address(es) to listen on;
 					# comma-separated list of addresses;
 					# defaults to 'localhost'; use '*' for all
 					# (change requires restart)
-#port = 5432				# (change requires restart)
+port = 5432				# (change requires restart)
 # Note: In RHEL/Fedora installations, you can't set the port number here;
 # adjust it in the service file instead.
 max_connections = 100			# (change requires restart)